contracts/governnace/GovernorAlpha.sol

```diff
@@ -0,0 +1,386 @@
+pragma solidity ^0.5.16;
+pragma experimental ABIEncoderV2;
+
+contract GovernorAlpha {
+    /// @notice The name of this contract
+    string public constant name = "Olympus Governor Alpha";
```

Comment:
Should this be "Olympus Governor OHMega" ? 😎

Reply:
yeeep